src/main/resources/defaultProperties/WebCapabilities.properties

```diff
@@ -10,6 +10,7 @@ mobileEmulation.width=
 mobileEmulation.height=
 mobileEmulation.pixelRatio=
 mobileEmulation.userAgent=
+BaseURL=https://demoqa.com/
```

Comment:
can we convert to lower camel case please?
we also need to expose this property in the configuration manager :)